### PR TITLE
Creation of audit and aws schema

### PR DESCRIPTION
**Pull request by LucioDonda**
|Related issue|
|---|
|#16322|

## Description
In order to use the new schema script in this PR the schemas needed for AWS and audited decoders are added

## Task
- [x] Create audit scheme
- [x] Create AWS schema
- [x] Update Documentation
- [x] QA validate

### CODE REVIEW

**Comment by JcabreraC** [comment]
Beware, the archives:
- src/engine/extension/schema/aws/fields.yml
- src/engine/extension/elasticsearch/7.x/wazuh-template.json
- src/engine/ruleset/schemas/wazuh-logpar-types.json

say they have completely changed because the tabulation has been changed. Change it to show only the PR changes.

**Review by JcabreraC — CHANGES_REQUESTED** [review]
The `src/engine/extension/schema/auditd/fields.yml` file is missing fields that decode the decoders.

**Comment by LucioDonda** [comment]
Auditd fields analysis:

- Fields used as temporary inside of the decoder:

auditd.log

- Fields renamed inside of the decoder:

auditd.log.acct
auditd.log.arch
auditd.log.comm
auditd.log.cwd
auditd.log.exe

- Fields saved in another:

auditd.log.cmd
auditd.log.src
auditd.log.record_type

- Deleted at the end of decoders:

auditd.log.copy
auditd.log.epoch
auditd.log.sub_kv
auditd.log.kv


- Added fields (moved from `.log.` ):

auditd.data.apparmor
auditd.data.cgroup
auditd.data.file
auditd.data.device
auditd.data.dir
auditd.data.grp
auditd.data.invalid_context
auditd.data.new-chardev
auditd.data.new-disk
auditd.data.new-fs
auditd.data.new-net
auditd.data.new-rng
auditd.data.ocomm
auditd.data.old-chardev
auditd.data.old-disk
auditd.data.old-fs
auditd.data.old-net
auditd.data.old-rng
auditd.data.success
auditd.data.vm
auditd.data.watch

- New fields (taken from hex script) :

auditd.log.node -> ~auditd.log.node
auditd.log.denied_mask
auditd.log.new_group
auditd.log.info
auditd.log.operation
auditd.log.proctitle
auditd.log.profile
auditd.log.requested_mask
auditd.log.root_dir
auditd.log.sw
auditd.log.data
auditd.log.path

File: src/engine/ruleset/decoders/auditd/auditd-kv.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ parse:
 normalize:
   - map:
       - auditd.log: +ef_merge/$~kv_map
-      - auditd.log.node: +ef_rename/$~node
+      - ~auditd.log.node: +ef_rename/$~node
```
Comment (nmkoremblum) [inline src/engine/ruleset/decoders/auditd/auditd-kv.yml:29]: Why has this field been changed to a temporal one?

Comment (LucioDonda) [reply src/engine/ruleset/decoders/auditd/auditd-kv.yml:29]: Going back to the pipeline, taken as a base for the decoder, that field is used and mapped as in this case, but I couldn't find any description of it or any other use in the schemas. IMHO It wasn't a wrong approach to define it as temporary, but if it is needed after, we could create a custom field with that name.

**Review by nmkoremblum — APPROVED** [review]
LGTM :+1:

**Review by juliancnn — APPROVED** [review]
LGTM!, only just need to fix sophos
```
    engine: "/home/vagrant/engine/wazuh/src/engine/ruleset/decoders/sonicwall/sonicwall-firewall-generated.yml" ==> An error occurred while trying to validate "decoder/sonicwall-firewall-generated/0": Parse stage: Building asset "decoder/sonicwall-firewall-generated/0" failed: Error building parser "logpar": An error occurred while parsing a log: Field 'observer.egress.interface.name' not found in schema
```

**Review by nbertoldo — APPROVED** [review]
LGTM!

**Review by JavierBejMen — APPROVED** [review]

**Review by JcabreraC — APPROVED** [review]
LGTM !

**Pull request merged**